delta/sdk-testkit/src/test/scala/ch/epfl/bluebrain/nexus/delta/sdk/testkit/AclsBehaviors.scala

```diff
@@ -70,15 +71,36 @@ trait AclsBehaviors {
   val anyOrg              = AnyOrganization(false)
   val anyOrgWithAncestors = AnyOrganization(true)
 
-  def create: Task[Acls]
+  def create: Task[(Acls, Permissions)]
 
   "An ACLs implementation" should {
-    val acls =
+    val (acls, permissions) =
       create
         .timeoutWith(10.seconds, new TimeoutException("Unable to create a permissions instance"))
         .memoizeOnSuccess
         .accepted
 
+    "return the full permissions for Anonymous if no permissions are defined" in {
```

Comment:
We should test again that the permissions for Anonymous are gone when we add some acls